ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/StandardRuleSetProvider.kt

```diff
@@ -36,6 +36,7 @@ class StandardRuleSetProvider : RuleSetProvider {
         SpacingAroundOperatorsRule(),
         SpacingAroundParensRule(),
         SpacingAroundRangeOperatorRule(),
+        SpacingInUnaryExpressionRule(),
```

Comment:
Can you move this to the experimental ruleset, so we indicate that the rule is incubating? We can't just put new rules to the standard ruleset unfortunately, as they need some battle-testing beforehand.